cadence/scripts/get_all_nfts_and_views_in_account.cdc

```diff
@@ -24,8 +25,8 @@ pub struct NFTCollectionData {
     }
 }
 
-pub fun main(ownerAddress: Address): {String: {String: AnyStruct}} {
-    let account = getAuthAccount(ownerAddress)
+access(all) fun main(ownerAddress: Address): {String: {String: AnyStruct}} {
+    let account = getAuthAccount<auth(Storage,BorrowValue, IssueStorageCapabilityController, PublishCapability, SaveValue, UnpublishCapability) &Account>(ownerAddress)
```

Comment:
It looks like the only thing you use the account object for here is issuing a capability and publishing a capability. Not sure if you need all the other entitlements. This applies to other scripts and transactions that you've modified here too, so I would look through them and make sure you have the minimal set of entitlements on each one